lib/senkyoshi/models/attachment.rb

```diff
@@ -10,11 +10,13 @@ def iterate_xml(xml, pre_data)
         @module_type,
         @files.first.name,
         @url,
+        @indent,
+        @file_name,
       ).canvas_conversion
       self
     end
 
-    def canvas_conversion(course, _resource)
+    def canvas_conversion(course, _)
```

Comment:
Let's avoid naming just `_` and keep it with a name so it is clear what would have been passed in.